geomesa-quickstart-lambda/src/main/java/com/example/geomesa/lambda/LambdaQuickStart.java

```diff
@@ -0,0 +1,223 @@
+/***********************************************************************
+ * Copyright (c) 2016-2017 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, Version 2.0
+ * which accompanies this distribution and is available at
+ * http://www.opensource.org/licenses/apache2.0.php.
+ ***********************************************************************/
+
+package com.example.geomesa.lambda;
+
+import org.apache.commons.cli.BasicParser;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.HelpFormatter;
+import org.apache.commons.cli.MissingOptionException;
+import org.apache.commons.cli.OptionBuilder;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.ParseException;
+import org.geotools.data.DataStoreFinder;
+import org.geotools.data.Transaction;
+import org.geotools.data.simple.SimpleFeatureWriter;
+import org.geotools.factory.Hints;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.locationtech.geomesa.accumulo.data.AccumuloDataStore;
+import org.locationtech.geomesa.lambda.data.LambdaDataStore;
+import org.locationtech.geomesa.lambda.data.LambdaDataStoreFactory;
+import org.locationtech.geomesa.lambda.data.LambdaDataStoreFactory$Params$Accumulo$;
+import org.locationtech.geomesa.lambda.data.LambdaDataStoreFactory$Params$Kafka$;
+import org.locationtech.geomesa.utils.geotools.SimpleFeatureTypes;
+import org.opengis.feature.simple.SimpleFeature;
+import org.opengis.feature.simple.SimpleFeatureType;
+import org.opengis.filter.Filter;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.PrintStream;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Random;
+
+public class LambdaQuickStart implements Runnable {
+
+  // reads and parse the command line args
+  @SuppressWarnings("AccessStaticViaInstance")
+  public static Map<String, String> parseOptions(String[] args) {
+    Options options = new Options();
+
+    options.addOption(OptionBuilder.hasArg()
+                                   .withArgName("brokers")
+                                   .withDescription("The comma-separated list of Kafka brokers, e.g. localhost:9092")
+                                   .isRequired()
+                                   .withLongOpt("brokers")
```

Comment:
hmm, flipping them makes it take two dashes for the short command:
```
-brokers,--b <brokers>
```
Even though it reports the short name, it prints out the full usage so it's pretty easy to see what's missing, no?